Dockerfile

```diff
@@ -18,117 +20,129 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     curl \
     kdiff3 \
     git \
-    vim # for debug
+    vim \
+  && rm -rf /var/lib/apt/lists/*
```

Comment:
You are going to see `  && rm -rf /var/lib/apt/lists/*` a lot (after any `apt-get` call). This is to wipe out its index so it is not saved to the image, which is where we are getting a decent chunk (about 800 MB) of the size savings.